release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -1301,6 +1301,22 @@ Targeting {product-title} 4.15, the OpenShift SDN CNI network plugin will not be
 [id="ocp-4-14-cloud-cred-operator-bug-fixes"]
 ==== Cloud Credential Operator
 
+* Previously, the Cloud Credential Operator utility (`ccoctl`) used an incorrect Amazon Resource Names (ARN) prefix for AWS GovCloud (US) and AWS China regions. The incorrect ARN prefix caused the `ccoctl aws create-all` command that is used to create AWS resources during installation to fail. This release updates the ARN prefixes to the correct values. (link:https://issues.redhat.com/browse/OCPBUGS-13549[*OCPBUGS-13549*])
+
+* Previously, security changes to Amazon s3 buckets caused the Cloud Credential Operator utility (`ccoctl`) command that is used to create AWS resources during installation (`ccoctl aws create-all`) to fail. With this release, the `ccoctl` utility is updated to reflect the Amazon s3 security changes. (link:https://issues.redhat.com/browse/OCPBUGS-11671[*OCPBUGS-11671*])
+
+[discrete]
+[id="ocp-4-14-cluster-version-operator-bug-fixes"]
+==== Cluster Version Operator
+
+* Previously, the Cluster Version Operator did not reconcile SecurityContextConstraints resources as expected. Cluster Version Operator now properly reconciles SecurityContextConstraints resources towards the state defined in the release image, reverting any unsupported modifications to them.
++
+Users who want to upgrade from earlier OCP versions and who operate workloads depending on modified system SecurityContextConstraints resources need to follow the procedure in https://access.redhat.com/solutions/7033949 to make sure their workloads are able to run without modified system SecurityContextConstraints. (link:https://issues.redhat.com/browse/OCPBUGS-19465[*OCPBUGS-19465*])
```

Comment:
Probably aware of this, but reminder that https://access.redhat.com/solutions/7033949 is currently in Unpublished state (i.e., not visible to customers) and should be published before 4.14 GA.